docs/deployment/file-access-auth.md

```diff
@@ -0,0 +1,159 @@
+			<key>Options</key>
+			<dict>
+				<key>AllowReadAccess</key>
+				<false/>
+				<key>AuditOnly</key>
+				<true/>
+			</dict>
+			<key>Processes</key>
+			<array>
+				<dict>
+					<key>BinaryPath</key>
+					<string>/usr/local/bin/my_foo_writer</string>
+					<key>TeamID</key>
+					<string>ABCDEF1234</string>
+				</dict>
+			</array>
+		</dict>
+	</dict>
+</dict>
+</plist>
+```
+
+## Details
+
+### Rule Matching
+
+When an operation occurs on a path that matches multiple configured path globs, the rule containing the "most specific" matching path glob is applied (i.e., the longest matching path).
+
+For example, consider the configured rules and paths:
+```
+RULE_1: /tmp/foo     [IsPrefix=true]
+RULE_2: /tmp/foo.txt [IsPrefix=false]
+RULE_3: /tmp         [IsPrefix=true]
+```
+The following table demonstrates which rule will be applied for operations on a given path:
+| Operation Path   | Rule Applied | Reason |
+| ---------------- | ------------ | ------ |
+| /tmp/foo         | `RULE_1`     | Matches prefix, more specific than `RULE_3` |
+| /tmp/foo/bar     | `RULE_1`     | Matches prefix, more specific than `RULE_3` |
+| /tmp/bar         | `RULE_3`     | Matches prefix |
+| /tmp/foo.txt     | `RULE_2`     | Matches literal, more specific than `RULE_1` |
+| /tmp/foo.txt.tmp | `RULE_1`     | Matches prefix, more specific than `RULE_3`, literal match doesn't apply |
+| /foo             | N/A          | No rules match operations on this path |
+
+If a configuration contains multiple rules with duplicate configured paths, only one rule will be applied to the path. It is undefined which configured rule will be used. Administrators should take care not to define configurations that may have duplicate paths.
+
+### Path Globs
+
+Configured path globs represent a point in time. That is - path globs are expanded when a configuration is applied to generate the set of monitored paths. Currently this is not a "live" representation of the filesystem. For instance, if a new file or directory is added that would match a glob after the configuration is applied, it is not immediately monitored.
+
+Within the main Santa configuration, the `FileAccessPolicyUpdateIntervalSec` key controls how often any changes to the configuration are applied as well as re-evaluating configured path globs to match the current state of the filesystem.
+
+### Prefix and Glob Path Evaluation
+
+Combining path globs and the `IsPrefix` key allows configurations greater control over the paths rules should match. Consider the configured path globs:
+```
+PG_1: /tmp/*         [IsPrefix = false]
+PG_2: /tmp/*         [IsPrefix = true]
+PG_3: /tmp/          [IsPrefix = true]
+PG_4: /tmp/file1.txt [IsPrefix = false]
+```
+
+And a filesystem that contains:
+```
+/
+	tmp/
+		file1.txt
+		file2.txt
+		dir1/
+			d1_f1.txt
+			d1_f2.txt
+```
+
+Now assume the configuration is applied, and moments later a new file (`/tmp/file3_new.txt`) new directory (`/tmp/dir2_new`) are both created:
```

Comment:
```suggestion
Now, assume the configuration is applied, and moments later a new file (`/tmp/file3_new.txt`) and a new directory (`/tmp/dir2_new`) are both created:
```

Reply:
Check the meaning here, it looked as if the conjunction and article were missing, but I might be misunderstanding the sense here.